optuna/integration/cma.py

```diff
@@ -483,7 +483,9 @@ def _to_optuna_params(
         return cma_param_value
 
 
-@deprecated("2.0.0", text="This class is renamed to :class:`~optuna.integration.PyCmaSampler`.")
+@deprecated(
+    "2.0.0", "4.0.0", text="This class is renamed to :class:`~optuna.integration.PyCmaSampler`."
```

Comment:
I'm wondering if it would be better to `@deprecated` calling in one line.
(It would help when someone want to get a list of deprecated features)

So, how about defining `_cmaes_deprecated_msg` like https://github.com/optuna/optuna/blob/efe6328f17e3c151ecb9752a637a551078606061/optuna/integration/chainermn.py#L27?

Reply:
Thanks. It is much clearer.
I applied your suggestion.